specificatie/openapi.yaml

```diff
@@ -132,16 +130,16 @@ paths:
           required: false
           schema:
             type: string
-            maxLength: 10
+            pattern: ^[a-zA-Z0-9À-ÿ' '\-\']{1,10}$
```

Comment:
niet gelijk aan de pattern in de request body. De ' voor en achter de spatie kan weg. Hier is voorvoegsel gedefinieerd als een string van min 1 en max 10 karakters. In de request body als een string van min 1 en max oneindig karakters

Reply:
Aangepast